srcpkgs/python3-jupyter_server/template

```diff
@@ -14,7 +14,7 @@ maintainer="dkwo <nicolopiazzalunga@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://jupyter.org"
 distfiles="${PYPI_SITE}/j/jupyter-server/jupyter_server-${version}.tar.gz"
-checksum=5fb5a219385338b1d13a013a68f54688b6a69ecff4e757fd230e27ecacdbf212
+checksum=9e3e9717eea3bffab8cfb2ff330011be6c8bbd9cdae5b71cef169fcece2f19d3
 # Tarball defines no tests
 make_check=no
```

Comment:
in which case, I get:
```
=> python3-jupyter_server-1.13.5_1: running do_check ...
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
/usr/lib/python3.10/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
  warnings.warn(
WARNING: The wheel package is not available.
/usr/bin/python3: No module named pip
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmprp9eg75u', '--quiet', 'argon2-cffi']' returned non-zero exit status 1.
=> ERROR: python3-jupyter_server-1.13.5_1: do_check: 'python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
=> ERROR:   in do_check() at common/build-style/python3-module.sh:40
```

Reply:
That's because argon2-cffi is in install_requires, but it's actually an optional dependency. We can either package it and add it to checkdepends or use make_check=no and add a comment `# needs python3-argon2-cffi which is not packaged`

Reply:
after adding `python3-argon2`, as well as a ton of other `checkdepends`, it still fails with
```
=> python3-jupyter_server-1.13.5_1: running do_check ...
============================= test session starts ==============================
platform linux -- Python 3.10.3, pytest-6.2.5, py-1.10.0, pluggy-0.13.1
rootdir: /builddir/jupyter_server-1.13.5, configfile: pyproject.toml, testpaths: jupyter_server/
plugins: mock-3.5.1, anyio-3.5.0, asyncio-0.14.0, timeout-1.4.2
timeout: 300.0s
timeout method: thread
timeout func_only: False
collected 0 items / 1 error

==================================== ERRORS ====================================
________________________ ERROR collecting test session _________________________
Defining 'pytest_plugins' in a non-top-level conftest is no longer supported:
It affects the entire test suite instead of just below the conftest as expected.
  /builddir/jupyter_server-1.13.5/jupyter_server/conftest.py
Please move it to a top level conftest file at the rootdir:
  /builddir/jupyter_server-1.13.5
For more information, visit:
  https://docs.pytest.org/en/stable/deprecations.html#pytest-plugins-in-non-top-level-conftest-files
=========================== short test summary info ============================
ERROR 
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.10s ===============================
```
What they actually require:
```
test =
    coverage
    pytest>=6.0
    pytest-cov
    pytest-mock
    pytest-timeout
    requests
    pytest-tornasync
    pytest-console-scripts
    ipykernel
```

Reply:
ok, after updating, error message is clear: we are missing pytest_tornasync